src/utils/privileges.cpp

```diff
@@ -149,6 +152,7 @@ static void security_drop_root_from_sudo() {
 		_fact("setuid error");
 		throw std::system_error(std::error_code(errno, std::system_category()));
 	}*/
+	home_dir = pw->pw_dir;
```

Comment:
UsePtr( pw->pw_dir )
const char* could be in theory a nullptr

when I add UsePtr for any T*
todo